src/api.js

```diff
@@ -2,14 +2,10 @@ import axios from 'axios';
 
 const BASE_URL = 'https://project-news-liliia.onrender.com/api';
 
-export const fetchArticles = async () => {
-  try {
-    const response = await axios.get(`${BASE_URL}/articles`);
-    return response.data.articles;
-  } catch (error) {
-    console.error('Error fetching articles:', error);
-    throw error;
-  }
+
+export const fetchArticles = () => {
+  return axios.get(`${BASE_URL}/articles`)
+    .then((response) => response.data.articles);
```

Comment:
I would recommend reading the notes on Axios for the recommended way to do this:

l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance